### DISCUSSION TITLE
Tests randomly fail and sometimes all pass

### DISCUSSION THREAD

**Discussion by ven7ura** [pestphp/pest#991] [Q&A]
I have recently installed `Laravel Breeze` to a proyect that already had `filamentphp` installed. I organized all the files and tests so that they use their proper `TestCase::class`.

Now running `pest` on the terminal I get random errors on certain runs. The errors keep jumping from file to file and sometimes I dont get any failed tests. These are some of the errors I see:

``` bash
ViewException ... Undefined array key 0 at vendor/laravel/framework/src/Illuminate/View/Concerns/ManagesComponents.php:124
```

``` bash
ViewException ... Failed to open stream: No such file or directory (View: .../resources/views/livewire/profile/update-password-form.blade.php) ... at vendor/livewire/livewire/src/Mechanisms/ExtendBlade/ExtendedCompilerEngine.php:37
```

``` bash
Expected response status code [>=200, <300] but received 500. ParseError: syntax error, unexpected end of file, expecting "elseif" or "else" or "endif" in .../storage/framework/views/b38b543530fefd540c3e583b3c780169.php:195
```

If I run `php artisan test` the test `pass` all the time. I suspect it has something to do with parallel testing, but I am not 100% sure. The errors I mentioned keep jumping from file to file, so it's not always the same affected PHP file.

**Comment by MickProPay** [comment]
I would love to get an answer on this as sometime my tests also pass and then out of the blue one would fail and then the very next run it passes....

**Comment by NathanLochala** [comment]
@MickProPay Have you gotten any resolution on this issue. I'm facing the same thing, except it is when using Pest/Livewire to test Filamentphp tables. Sometimes the tests pass, sometimes they fail, no clear correlation.
